types/stable/vim.lua

```diff
@@ -16,6 +16,8 @@ vim.keymap = require("vim.keymap")
 vim.lsp = require("vim.lsp")
 vim.treesitter = require("vim.treesitter")
 vim.treesitter.highlighter = require("vim.treesitter.highlighter")
+vim.treesitter.query = require("vim.treesitter.query")
+vim.treesitter.language = require("vim.treesitter.langauge")
```

Comment:
should be language

Reply:
no clue how I did that but should be fixed now lol